src/page-multi-facility/MultiFacilityPage.tsx

```diff
@@ -1,38 +1,35 @@
 import React from "react";
 import styled from "styled-components";
 
-import PromoBoxWithButton from "../design-system/PromoBoxWithButton";
 import SiteHeader from "../site-header/SiteHeader";
+import ProjectionsHeader from "./ProjectionsHeader";
+import ScenarioSidebar from "./ScenarioSidebar";
 import ToggleRow from "./ToggleRow";
 
-const LeftColumn = styled.div`
-  width: 300px;
-`;
 const MultiFacilityPageDiv = styled.div``;
 
+const MultiFacilityImpactDashboard = styled.main.attrs({
+  className: `
```

Comment:
This is more of an "FYI" to anyone who is interested in mixing Tailwind classes into a styled component